### DISCUSSION TITLE
什么时候可以修复这个问题集群问题

### DISCUSSION THREAD

**Discussion by zhenggexia** [apache/pekko#1328] [Q&A]
akka上有提到过一个问题：https://github.com/akka/akka/issues/31095
集群问题导致所有节点被关闭，这个有计划修复吗？预计何时修复

**Comment by He-Pin** [comment]
我在工作中只用了流，没有用集群，因为这个是个开源且自愿的项目，所以没有许诺。但是欢迎贡献。

**Comment by pjfanning** [comment]
Closing - as we have https://github.com/apache/pekko/issues/578 already
